bai-tap-buoi-9/main.js

```diff
@@ -1 +1,151 @@
-const 
\ No newline at end of file
+// Hàm kiểm tra số nguyên
+function checkNumber(number) {
+    if (parseInt(number)==number) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+// Bài 1:
+const realSalary = (money) => {
+    if (money >= 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.3)
+        console.log('Lương ròng: '+money*0.7)
+    }
+    else if (money >= 7000000 && money < 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.2)
+        console.log('Lương ròng: '+money*0.8)
+    }
+    else if (money >= 0 && money < 7000000) {
+        console.log('Thuế thu nhập: ' + money*0.1)
+        console.log('Lương ròng: '+money*0.9)
+    }
+    else {
+        console.log('Nhập đúng tiền lương')
+    }
+}
+
+// Bài 2:
+const ageSchool = (age) => {
+    if (checkNumber(age) && age >= 16) {
+        console.log('Bạn đã đủ tuổi vào lớp 10')
+    }
+    else if(checkNumber(age) && age<16) {
+        console.log('Bạn chưa đủ tuổi vào lớp 10')
+    }
+    else{
+        console.log('Mời bạn nhập số nguyên')
+    }
+}
+// Bài 3:
+const equalsNumber = (number) => {
+    if(checkNumber(number) && number>100) {
+        console.log('Số '+number+' lớn hơn 100')
+    }
+    else if(parseInt(number)==number && number==100) {
+        console.log('Hai số bằng nhau')
+    }
+    else if(parseInt(number)==number && number<100) {
+        console.log('Số '+number+' nhỏ hơn 100')
+    }
+    else {
+        console.log('Mời bạn nhập số nguyên')
+    }
+}
+
+// Bài 4: 
+const findMax = (a,b,c) =>{
+    var flag=0
+    var max
+    if(checkNumber(a)&& checkNumber(b) && checkNumber(c)){
+        flag = 1;
+        max =a;
+    }
+    else {
+        console.log('Hãy nhập số nguyên')
+    }
+    if (b > max) {
+        max = b
+    }
+    if (c > max ) {
+        max = c
+    }
+    if (flag == 1) {
+        console.log('Số lớn nhất là: ' + max)
+    }
+}
+
+// Bài 5: 
+const rankStudent = (point) => {
+    if(point >=9 && point <=10) {
+        console.log('Xếp hạng A')
+    }
+    else if(point >=7 && point <9) {
```

Comment:
```suggestion
    } else if (point >=7) {
```
Bên trên đã kiểm tra >=9 rồi, thì xuống dưới này chắc chắn phải < 9, chú ý nhìn các dấu cách